networklibrary/src/main/java/com/eis/communication/network/commands/QuitNetwork.java

```diff
@@ -0,0 +1,33 @@
+package com.eis.communication.network.commands;
+
+import androidx.annotation.NonNull;
+
+import com.eis.communication.Peer;
+import com.eis.communication.network.NetSubscriberList;
+
+/**
+ * Command to remove a peer from the subscribers
```

Comment:
Not in line with the `execute` method: I've been struggling for a couple of seconds to understand what this command actually does if it removes somebody else or the current user.

Reply:
Oh you're right, we changed what the command does and never updated this comment, fixing 👍